content/get-haiku/r1beta2/release-notes.md

```diff
@@ -0,0 +1,236 @@
+
+*To download Haiku or learn how to upgrade from R1/beta1, see "[Get Haiku!](/get-haiku/)". For press inquiries, see "[Press contact](#press-contact)".*
+
+## System requirements
+This release is available on the x86 32-bit platform, as well as the x86_64 platform. Note that BeOS R5 compatibility is only provided on the 32-bit images.
+
+<table><tr><td>
+<h4>MINIMUM (32-bit)</h4>
+<ul>
+<li><strong>Processor</strong>: Intel Pentium II; AMD Athlon</li>
+<li><strong>Memory:</strong> 256MB</li>
+<li><strong>Monitor:</strong> 800x600</li>
+<li><strong>Storage:</strong> 3GB</li>
+</ul>
+</td><td>
+<h4>RECOMMENDED (64-bit)</h4>
+<ul>
+<li><strong>Processor</strong>: Intel Core i3; AMD Phenom II</li>
+<li><strong>Memory:</strong> 2GB</li>
+<li><strong>Monitor:</strong> 1366x768</li>
+<li><strong>Storage:</strong> 16GB</li>
+</ul>
+</td></tr></table>
+
+SSE2 support is required to use the WebPositive web browser. On machines where this is not available, it is recommended to install the NetSurf browser instead.
+
+## Highlights
+
+This release contains 20 months' worth of improvements. Below you find some of the highlights.
+
+### Improved HiDPI support
+
+Begun but significantly incomplete at the time of Beta 1, this release sees a significant advancement in support for HiDPI monitors (and UI scaling in general.) Unlike other operating systems which largely enact a "device pixel ratio", that converts a number of "internal pixels" to "screen pixels", Haiku instead scales everything based on font size, and leaves pixels to mean "screen pixels" in isolation. This also means that one can effectively change the UI size and scale on any display by changing the system font size, and everything else should adjust to match it.
+
+Below are two screenshots, showing ActivityMonitor and AboutSystem with a 12pt font size on the left (Haiku's default), and with an 18pt font size on the right (in other words, 150% of "normal.")
+
+<style>td>img{max-width:400px;}</style>
+<table><tr><td><img src="/files/get-haiku/r1beta2/normal-activitymonitor-aboutsystem.png"></td><td><img src="/files/get-haiku/r1beta2/hidpi-activitymonitor-aboutsystem.jpg"></td></tr></table>
+
+There are still some bugs to be ironed out (such as scrollbars not drawing with proper scaling applied, as seen in this screenshot), but things are now generally usable under HiDPI. Other applications, such as Deskbar and Tracker, saw significant improvements to their behavior on HiDPI, as well.
+
+### Deskbar improvements
+
+The Deskbar has had various improvements. It now has a mini-mode, where the icons on the shelf expand leftwards. You can activate it by dragging the shelf space by the right handle to the top of the screen.
+
+<img src="/files/get-haiku/r1beta2/deskbar_minimode.png"/><br>
+<small><i>New mini-mode to put the Deskbar at the top of the screen.</i></small>
+
+Furthermore, there are more options for showing larger icons, which is especially useful on screens with a higher DPI.
```

Comment:
Was the slider already in Beta1? Instead, maybe point to the improved "Auto-raise" that now also "auto-lowers" when the mouse leaves the Deskbar area. The screenshot below would become unnecessary.